### PR TITLE
Fixed thanks to help from Amanda Stouder

### LINKED ISSUE #6: Emergency Fix-- Missing angular.json
When merged into git, didn't realize that it didn't include angular.json or app.component.* 

**Acceptance Criteria:**

- [ ]  Can run ng commands inside of the folder again 
- [ ] Can serve the application and it'll show in browser

### PR DESCRIPTION
**Pull request by MCoriale**
Closes #6

### CODE REVIEW

**Review by hanseljt77 — APPROVED** [review]
Bless up to Amanda. Looks good!

**Comment by DRSisco** [comment]
Perfect. Yeah filling in the .gitignore (I'm lazy and use things like https://www.toptal.com/developers/gitignore) then calling `git add .` will easily fix this issue as you probably found out. 

Please note the purpose of reviews is to catch more uhhh obvious issues like this. I off-hand mentioned the missing files in https://github.com/RoseContactTracer/covid-tracer-frontend/pull/2 
Its a tad on me for making a long comment that lost attention of people. 

Basically you guys don't have a CI, so I would typically expect the reviewer to pull the branch down and run things and review it based on that. (maybe an exception on smaller changes)

**Comment by noahthomas88** [comment]
Looks good!
